src/ResourceManagement/KeyVaultManagement/Microsoft.Azure.Management.KeyVault/Generated/IKeyVaultManagementClient.cs

```diff
@@ -1,7 +1,3 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@kirthik thanks. The header parameter was not set. Fixed it and updated the commit. :)